### PR TITLE
Add development disclaimer and refs to projects and roadmap

### LINKED ISSUE #110: Update readme
The readme is massively outdated to say the least.
> and is currently not under active development.

### PR DESCRIPTION
**Pull request by bmagyar**
resolves #110 

FYI @AndyZe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ linux: [![Build Status](https://travis-ci.org/ros-controls/ros2_control.svg?bran
 
 ## Disclaimer
 
+`ros2_control` is under heavy development, to get an update of the project please either tag along to one of our working group meetings (see more on discourse) and/or check the resources below:
```
Comment (AndyZe) [inline README.md:8]: perhaps add the full link to discourse in case they aren't aware:  `discourse.ros.org`

File: README.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ linux: [![Build Status](https://travis-ci.org/ros-controls/ros2_control.svg?bran
...
+* Actively tracked projects can be found here: https://github.com/orgs/ros-controls/projects
+* Consult the roadmap here: https://github.com/ros-controls/roadmap
```
Comment (AndyZe) [inline README.md:10]: I think you should add the demo repo, too. But you are more knowledgeable about the state of the demos.

* See demos here (under heavy development): https://github.com/ros-controls/ros2_control_demos

File: README.md (hunk 1 above)
Comment (bmagyar) [reply README.md]: good point, added!

File: README.md (hunk 2 above)
Comment (bmagyar) [reply README.md:10]: good point, added!

**Comment by bmagyar** [comment]
I'll take that as everyone is happy

**Pull request merged**